src/ui/otb/OtbRound.ts

```diff
@@ -115,6 +132,9 @@ export default class OtbRound implements OtbRoundInterface, PromotingInterface {
       payload.fen = setupFen
     }
 
+    const clockType = settings.otb.clock.clockType() as ClockType
```

Comment:
I'd prefer have the clockType as a parameter of `startNewGame` instead of fetching localstorage.
This way you don't have to cast. And if arg is undefined it means it's a game without clock.

Reply:
Makes sense. Done.